src/fm/usage.cljc

```diff
@@ -367,10 +373,11 @@
 (defm echo-refined
   ^{:fm/args ::http-req
     :fm/ret  ::http-resp
-    :fm/rel  (fn [{:keys [args ret]}] ; same interface as `:fn` in `s/fdef`
+    :fm/rel  (fn [{[{:keys [body] :as req}] :args
```

Comment:
_Super_ nitpick since this is just example usage but I don't love having a binding laying around that isn't used (`req` in this case). Can leave if you want, just an observation.

Reply:
oh, im totally on your side with this (and i think kondo would yell at us for this too, in a good way). i left it in as a reference to the idea that the `:args` key destructures identically to the `fm` itself (or something along those lines), almost more as a reminder to myself. i think we nix it entirely once `:fm/rel` is implemented and we're thinking through good `fm.usage` examples for it.